src/test/pybind/test_rbd.py

```diff
@@ -12,7 +12,7 @@
 import sys
 
 from assertions import (assert_equal as eq, assert_raises, assert_not_equal,
-                        assert_greater_equal)
+                        assert_greater_equal, assert_greater)
```

Comment:
These are just for compatibility from when we used nose.  With pytest, plain `assert` can be used.